src/main/webapp/WEB-INF/tags/teammatesLogo.tag

```diff
@@ -0,0 +1,5 @@
+<%@ tag description="TEAMMATES logo brand"%>
+
+<a class="navbar-brand" href="/">
+    <img height="100%" src="../images/teammateslogo.jpg" alt="TEAMMATES[Logo] - Online Peer Feedback/Evaluation System for Student Team Projects">
```

Comment:
The test is failing because I haven't updated the test HTML files with GodMode. Will do so in a bit.